src/Xena.Contracts/ApiRoutes/LedgerLineRoutes.cs

```diff
@@ -8,8 +8,8 @@ public class LedgerLineRoutes : BaseRoutes
         /// <summary>"Multiple"</summary>
         public const string PostMultiple = "{ledgerId}/Multiple";
 
-        /// <summary>"KID/Multiple"</summary>
-        public const string PostMultipleFromKID = "{ledgerId}/KIDMultiple";
+        /// <summary>"MultipleBasic"</summary>
+        public const string PostMultiplePayment = "{ledgerId}/MultiplePayment";
```

Comment:
Normally we would do {ledgerId}/Payment/Multiple (as if there is an endpoint to post just one - it would be just {ledgerId}/Payment)